src/common/components/form/validationUtils.tsx

```diff
@@ -0,0 +1,57 @@
+import { formatMessage } from '../../translation/utils';
+import { newMoment } from '../../time/utils';
+import {
+  SUPPORTED_START_BIRTHDAY,
+  DEFAULT_DATE_FORMAT,
+} from '../../time/constants';
+
+/** validateBirthDay()
+ * Validate user input child birthday.
+ * This app only target recently born child, so we support child which born after 2019 only.
+ * Input time which come from future is not accepted
+ * @param value Input value.
+ */
+const validateBirthDay = (value: string | number | undefined) => {
+  if (!value) {
+    return formatMessage('validation.general.required');
+  }
+
+  const inputMoment = newMoment(value, DEFAULT_DATE_FORMAT);
+  const nowMoment = newMoment();
+
+  if (!inputMoment.isValid()) {
+    return formatMessage('validation.date.invalidFormat');
+  }
+
+  const supportedStart = newMoment(SUPPORTED_START_BIRTHDAY);
+
+  if (inputMoment < supportedStart || inputMoment > nowMoment) {
+    return formatMessage('validation.date.nonSupported');
```

Comment:
`notSupported` sounds more proper English like to my ear

Reply:
The word exists in the dictionary though:
https://en.wiktionary.org/wiki/nonsupported

and unsupported might be even better!

Reply:
Will do!